packages/joint-core/demo/routing/src/routing.js

```diff
@@ -105,4 +105,4 @@ $('.router-switch').on('click', function(evt) {
     }
 
     link.set('connector', { name: connector });
-});
+}, false);
```

Comment:
This is not necessary, `useCapture` is `false` by default

```suggestion
});
```